src/cool_seq_tool/sources/uta_database.py

```diff
@@ -378,6 +378,21 @@ async def validate_genomic_ac(self, ac: str) -> bool:
         result = await self.execute_query(query)
         return result[0][0]
 
+    async def validate_mane_transcript_acc(self, transcript_list: list[dict]) -> bool:
+        """Return whether or not a MANE transcript exists in UTA
```

Comment:
You should modify the description to state that it only checks the first element in `transcript_list`. You may also consider renaming `transcript_list` to `mane_transcripts`